core/domain/user_domain.py

```diff
@@ -0,0 +1,52 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects for user."""
+
+import feconf
+
+class UserGlobalPrefs(object):
+    """Domain object for user global email preferences"""
+
+    def __init__(self, site_updates, editor_role_notifications,
```

Comment:
Hmm, I wouldn't skimp on the full names here. The args passed in should exactly match the attrs on 'self' (by convention).

Also, please include docstrings.